src/test/features/response/form/models/howMuchOwed.ts

```diff
@@ -0,0 +1,101 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai'
+import * as randomstring from 'randomstring'
+import { Validator } from 'class-validator'
+import { expectValidationError } from '../../../../app/forms/models/validationUtils'
+import { HowMuchOwed } from 'features/response/form/models/howMuchOwed'
+import { ValidationErrors } from 'features/validationErrors'
+
+describe('HowMuchOwed', () => {
+  describe('constructor', () => {
+    it('should set the primitive fields to undefined', () => {
+      const howMuchOwed = new HowMuchOwed()
+      expect(howMuchOwed.amount).to.be.undefined
+      expect(howMuchOwed.text).to.be.undefined
+    })
+  })
+
+  describe('deserialize', () => {
+    it('should return an instance initialised with defaults for undefined', () => {
+      expect(new HowMuchOwed().deserialize(undefined)).to.eql(new HowMuchOwed())
+    })
+
+    it('should return an instance from given object', () => {
+      const description: string = 'I do not owe full amount'
+      const amount: number = 300
+      const result: HowMuchOwed = new HowMuchOwed().deserialize({
+        amount: amount,
+        text: description
+      })
+      expect(result.amount).to.be.equals(amount)
+      expect(result.text).to.be.equals(description)
```

Comment:
Would be nice if you could change these two to `.to.be.equal` to make it read correctly.